dev/MRTCore/packaging/Microsoft.ApplicationModel.Resources.PriGen.targets

```diff
@@ -221,7 +204,6 @@
     <AppxDefaultResourceQualifiers Condition="'$(TargetPlatformIdentifierAdjusted)' == 'Windows' and '$(TargetPlatformVersion)' == '8.2'">$(AppxDefaultResourceQualifiers_Windows_82)</AppxDefaultResourceQualifiers>
```

Comment:
we'd better check the version ahead of time, and never need this.

Reply:
I suspect AppxDefaultResourceQualifiers may not even be needed. I'll tackle that in the next round.